craco.config.js

```diff
@@ -76,7 +76,6 @@ module.exports = {
           project: "scroll-io",
           include: "./build",
           release: process.env.REACT_APP_VERSION,
-          authToken: process.env.REACT_APP_SENTRY_AUTH_TOKEN,
```

Comment:
@yqrashawn as said - if the frontend needs the token, then it MUST be exposed publicly. in this case, it seems like you completely remove the token. However, (the new version of) Sentry seems not to require a token at all, only the DSN (cf. https://docs.sentry.io/platforms/javascript/guides/react/ ). Can you confirm this works, and if so, then proceed to completely remove the auth_token from the code, envs/templates/docs, and netlify "secrets"?